### PR TITLE
45: provide access to Bundle to presenters

### PR DESCRIPTION
**Pull request by jshvarts**

### CODE REVIEW

File: app/src/main/kotlin/com/exallium/todoapp/mvp/BasePresenter.kt [hunk 1]
```diff
@@ -8,7 +8,7 @@ import rx.subscriptions.CompositeSubscription
  * The Presenter is considered "alive" when it is created, but "dead" as soon as
  * the underlying view is destroyed.
  */
-abstract class BasePresenter<out V : BaseView>(view: V) {
+abstract class BasePresenter<out V : BaseView>(private val view: V) {
```
Comment (exallium) [inline app/src/main/kotlin/com/exallium/todoapp/mvp/BasePresenter.kt]: we may as well just make this protected.

Comment (jshvarts) [reply app/src/main/kotlin/com/exallium/todoapp/mvp/BasePresenter.kt]: do you suggest making "view" protected here and removing "view" param from the constructors of the subclasses?

Comment (jshvarts) [reply app/src/main/kotlin/com/exallium/todoapp/mvp/BasePresenter.kt]: Can you help me out with this one?

Comment (exallium) [reply app/src/main/kotlin/com/exallium/todoapp/mvp/BasePresenter.kt]: @jshvarts yes, make the view param protected.  no need to set it more than once I think

**Review by exallium — APPROVED** [review]

**Pull request merged**